### ISSUE TITLE
Splitting training and validation data

### ISSUE THREAD

**Issue by nirajvermafcb** [fizyr/keras-maskrcnn#48]
How to train training and validation data and get both training and validation loss simultaneously.
